ztools/ztls/handshake_messages.go

```diff
@@ -1394,6 +1395,7 @@ func (m *newSessionTicketMsg) unmarshal(data []byte) bool {
 		return false
 	}
 
+	m.lifetimeHint = uint32(data[4])<<24 | uint32(data[5])<<16 | uint32(data[6])<<8 | uint32(data[7])
```

Comment:
There's a binary module, but I've seen it done both ways. Usually using the binary package isn't worth it unless you're doing a lot of marshaling. In this case, I don't think it matters and we're fine building our own int from bytes directly.